sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerConfig.java

```diff
@@ -66,6 +66,14 @@
   @JsonProperty
   private DateTimeZone sqlTimeZone = DateTimeZone.UTC;
 
+  @JsonProperty
+  private boolean metadataSegmentCacheEnable = false;
```

Comment:
Would you please add a doc for this configuration?

Reply:
sure, added